go.mod

```diff
@@ -85,6 +85,7 @@ require (
 	github.com/wercker/journalhook v0.0.0-20180428041537-5d0a5ae867b3
 	github.com/wsxiaoys/terminal v0.0.0-20160513160801-0940f3fc43a0 // indirect
 	go.opencensus.io v0.22.3
+	go.uber.org/multierr v1.1.0
```

Comment:
We use `https://github.com/hashicorp/go-multierror` elsewhere in the codebase. Any reason for the switch?

Reply:
For `multierr.Combine`. I'll revert this.